org.civicrm.reqangular/src/common/modules/directives.js

```diff
@@ -2,18 +2,21 @@ define([
   'common/angular',
   'common/decorators/ui-select/ui-select-choices',
   'common/decorators/angular-date/datepicker-addon',
+  'common/decorators/ui-bootstrap/uib-tabset',
+  'common/angularBootstrap',
   'common/ui-select',
   'common/modules/templates',
   'common/modules/controllers',
   'common/modules/services',
   'common/modules/apis'
-], function (angular, uiSelectChoicesCustomScrollBars, datepickerAddon) {
+], function (angular, uiSelectChoicesCustomScrollBars, datepickerAddon, uibTabset) {
```

Comment:
> Can you find out what common/modules/services and common/modules/apis are required for in this case? Why does this module needs to have access to them?

1. `common/modules/services`is not required - removed
2. `common/modules/apis`is required by https://github.com/civicrm/civihr/blob/e95e63f7836e7e20b163e87754dd0652c143f1af/org.civicrm.reqangular/src/common/directives/contact-actions/contact-actions.js which internally calls a Controller and that needs `api.contactActions`